packages/react-dropdown/src/index.js

```diff
@@ -86,7 +86,7 @@ const Dropdown = ({
   highlight = false,
   ...props
 }: Props) => {
-  if (defaultSelectedItems.length && selectedItems != null) {
+  if (defaultSelectedItems != null && selectedItems != null) {
```

Comment:
This makes sure the code doesn't throw type error if the user explicitly defines `defaultSelectedItems` to `null`

Reply:
okey